src/app/breadcrumbs/assessment-breadcrumb/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { memo, FC } from 'react';
+import { compose } from 'redux';
+
+import { getTranslateText as translate } from '../../../lib/translateText';
+import translations from '../../../lib/localization';
+
+import withAssessment, {
+  Props as AssessmentProps
+} from '../../../components/with-assessment';
+
+interface Props extends AssessmentProps {}
+
+const AssessmentBreadcrumb: FC<Props> = ({ assessment }) => (
+  <span>
+    {translate(assessment?.entity?.title ?? translations.breadcrumb.notFound)}
+  </span>
```

Comment:
- span er ikke nødvendig her
- translations.breadcrumb.notFound trenger ikke være inne i translate()

```suggestion
  <>
    {translate(assessment?.entity?.title) ?? translations.breadcrumb.notFound}
  </>
```

Reply:
6e389e8